test/specs/dist-default-types.spec.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs-extra');
+const sandbox = require('./sandbox');
+
+describe('distribute default-types', () => {
+
+    beforeAll(() => sandbox.init()
+        .then(() =>
+            sandbox.scaffold({
+                type: 'stream-api',
+                domain: 'acme.org',
+                author: 'coyote',
+                namespace: 'garden.pet',
+                shortName: 'garden-pet',
+                projectName: 'garden-pet-stream-api',
+                title: 'Garden Pet Stream API',
+                plugins: ['maven', 'dotnet'],
+                javaPackage: 'org.acme.garden.pet.v1',
+                mavenGroupId: 'org.acme.garden',
+                dotnetNamespace: 'Org.Acme.Garden.Pet',
+                dotnetPackageId: 'Org.Acme.Garden.Pet.StreamApi',
+                mqs: 'Kafka'
+            })));
+    afterAll(sandbox.clean);
+
+    it('should copy default-types in dist', () => sandbox.generate()
+        .then(({dir}) => fs.exists(`${dir}/dist/model/ts/node_modules/apikana/default-types.ts`, res => res || fail())))
```

Comment:
(HINT: Personal opinion ahead)
I suggest to not write too compact code. I would prefer readable, maintainable code. No need to manually make code compact. Minifiers got invented for this.

Cited:
> Any fool can write code that a computer can understand. Good programmers write code that humans can understand.
> Martin Fowler

Reply:
Hi Andreas, thank you for pointing this out, we are planning a refactoring to solve these kind of issues.